plugins/no_follow.rb

```diff
@@ -0,0 +1,32 @@
+# Jekyll Auto Nofollow Plugin
+# Automatically adds rel='external nofollow' to outgoing links.
+
+require 'jekyll'
+require 'nokogiri'
+
+module Jekyll
+    module NoFollow
+        def nofollow(content)
+            dom = Nokogiri::HTML.fragment(content)
+
+            # Find all links
+            dom.css('a').each do |link|
+
+                # All external links start with 'http', skip when this one does not
+                next unless link.get_attribute('href') =~ /\Ahttp/i
+
+                # Play nice with links that already have a rel attribute set, skip it
+                next if link.get_attribute('rel')
```

Comment:
I don't know about this, that means one can cheat our system

Reply:
Another take: It allows for exceptions and keeps the documented Jekyll behavior as is.

Reply:
Can we just append ' external nofollow` if it exists?

Reply:
Sounds like a plan

Reply:
Done.